layouts/shortcodes/swaggerui-api.html

```
@@ -0,0 +1,50 @@
+{{/*
+  Renders a Swagger UI viewer for an OpenAPI spec, loading its own assets.
+  No front matter type is required.
+  Usage:
+  {{< swaggerui src="example.json" >}}
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)